clients/python-wrapper/setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 NAME = "lakefs"
-VERSION = "0.1.0-alpha.1"
+VERSION = "0.1.0-alpha.3"
```

Comment:
guess there was alpha.2 :)

Reply:
Thanks - it's currently being released to the testpypi part of the experimenting with the release and documentation